buildSrc/src/main/java/com/google/firebase/gradle/plugins/PublishingPlugin.kt

```diff
@@ -191,9 +191,9 @@ abstract class PublishingPlugin : Plugin<Project> {
     allFirebaseLibraries: List<FirebaseLibraryExtension>
   ): ReleaseMetadata? {
     val projectsToPublish = project.provideProperty<String>("projectsToPublish").orNull
```

Comment:
I thought about that, but decided otherwise for two reasons:

1.) The linter moves it to a newline, which breaks the organization a tiny bit (very nit)
2.) The ideology is different than excepted, especially considering the one directly below it (`releaseName`). Instead of creating a variable and specifying an alternative value, it has a side effect of returning.

thoughts?

Reply:
1) If you think it looks too ugly, sgtm
2) `return` as a right-hand value of an elvis operator is OK. It's specifically called in the [docs about elvis operator.](https://kotlinlang.org/docs/null-safety.html#elvis-operator)